README.md

```diff
@@ -66,3 +70,41 @@ To add some config settings (see [http://supervisord.org/configuration.html]()):
           section: supervisord
           value: 2048
 ```
+
+To stop this role from starting Supervisor at all the
+sansible_supervisor_start_on_boot flag is available, useful if you want to start
+once a service has been added:
+
+```YAML
+- name: Install and configure supervisor with a service
+  hosts: "somehost"
+
+  roles:
+    - role: sansible.supervisor
+      sansible_supervisor_start_on_boot: no
+
+  post_tasks:
+    - name: Add service definition
+      become: yes
+      file:
+        dest: "{{ sansible_supervisor_conf_dir }}/service.conf"
+        src: service.conf
+
+    - name: Ensure supervisor service is started
+      become: yes
+      service:
+        name: supervisor
+        state: started
+
+    - name: Ensure service is present
+      become: yes
+      supervisorctl:
```

Comment:
Should this and below be

```yaml
service:
  name: supervisorctl
```
?

Reply:
I am ensuring that a supervisor controlled service is present, hence the use the of the supervisorctl module.

Reply:
Just making sure because of the `name: service`.  Approved.

Reply:
Renamed to my_app since it was a bit confusing.